src/roiextractors/testing.py

```diff
@@ -203,7 +203,11 @@ def check_segmentations_equal(
     ) == set(
         segmentation_extractor2.get_roi_pixel_masks(roi_ids=segmentation_extractor1.get_roi_ids()[:1])[0].flatten()
     )
-    assert_array_equal(segmentation_extractor1.get_image(), segmentation_extractor2.get_image())
+    for image_name in segmentation_extractor1.get_images_dict().keys():
+        assert_array_equal(
```

Comment:
Wouldn't `assert_array_almost_equal` be more appropriate?

Reply:
Why is that?

Reply:
If the dtype of the images end up being float their specific floating point representation might different even if they are same value. Not sure how robust is `assert_array_equal` by itself.

Reply:
Try something like this
```python
In [13]: x = np.random.rand(10, 10)

In [14]: np.testing.assert_array_equal(x, x.round(22))
```

Reply:
If strict equality passes without issue, why would we lessen it? There should in principle be no data loss even at some arbitrary level of rounding, and it would even be useful to know if such a event can occur

Reply:
The role of this function is to test if two segmentations are equal. The fact that it passes the tests doesn't mean that it will work "in the field" . This is like comparing floats with equality because it works on the test set.

Reply:
In other words you avoid false positives for the assertion.